pmd-core/src/main/java/net/sourceforge/pmd/PMD.java

```diff
@@ -504,6 +504,7 @@ public static int run(String[] args) {
                 final TimingReportRenderer renderer = new TextTimingReportRenderer();
                 try {
                     // Don't close this writer, we don't want to close stderr
+                    @SuppressWarnings("PMD.CloseResource")
```

Comment:
interestingly, as we whitelist when the autoclosable is passed as an argument, we may want to ignore `System.err`, `System.in` and `System.out`